src/server/scripts/EasternKingdoms/ZulGurub/boss_gahzranka.cpp

```diff
@@ -82,12 +82,12 @@ class boss_gahzranka : public CreatureScript // gahzranka
                 switch (eventId)
                 {
                     case EVENT_FROSTBREATH:
-                        DoCastVictim(SPELL_FROSTBREATH, true);
-                        events.ScheduleEvent(EVENT_FROSTBREATH, urand(7000, 11000));
+                        DoCastVictim(SPELL_FROSTBREATH, false);
```

Comment:
it's false by default, the parameter can be supressed

Reply:
Yep, I just checked the ac dir for some examples. Back then we used to have it even on default non-triggered spells.
insert garrosh meme: times changes.jpg